docs/examples/keda/README.md

```diff
@@ -55,7 +55,7 @@ the `CloudPubSubSource` scalable. Note that you could do this for any of the
 
    ```shell
    gcloud projects add-iam-policy-binding $PROJECT_ID \
-     --member=serviceAccount:cre-dataplane@$PROJECT_ID.iam.gserviceaccount.com \
+     --member=serviceAccount:events-sources-gsa@$PROJECT_ID.iam.gserviceaccount.com \
```

Comment:
The `-gsa` suffix seems like an unnecessary stutter to me. GSAs in APIs are rarely referred to by name alone (they usually have the domain attached), so I'm not sure when they'd be confused with any other type of identity (like KSA). I'm not an expert so I could be missing an ambiguous case.

Reply:
I probably don't have enough experience with these systems to fully understand your concern. I'm sympathetic to desire to avoid stutter, but I would imagine myself referring to this in most conversations as "events-sources-gsa" in general, not as "event-sources@PROJECT_ID.iam.gserviceaccount.com" (And hence I like the -gsa suffix for additional clarity.)

Reply:
Thanks for the feedback! I decided to keep the suffix.